src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -96,6 +96,7 @@ private void testGeneralQuestionOperations() throws Exception {
 
         testEditQuestionLink();
         testEditQuestionAction();
+        testEditVisibilityAction();
```

Comment:
Do I need to change the name of the method to narrow it down to giver's team members and giver?